Cartfile

```diff
@@ -13,3 +13,4 @@ github "st3fan/SWXMLHash"           "xcode-7.0"        # So we get Swift 2.0 fix
 github "DaveWoodCom/XCGLogger"      "swift_2.0"        # Previously: swift_1.2
 github "sleroux/google-breakpad-ios" "master"
 github "adjust/ios_sdk"             "v4.3.0"
+github "AgileBits/onepassword-extension" "add-framework-support"
```

Comment:
@st3fan @jhugman, the [App Extension API](https://github.com/AgileBits/onepassword-app-extension) supports Carthage on the `add-framework-support`. Which is being kept in sync with master. We strongly recommend that you point to its latest commit, `8afe2fad841c0439c4db6f8e2ac4d4c7e1e7a4bd` (the merge commit right after the `1.6.4`) or to the `1.6.4` tag itself.